code/game/gamemodes/changeling/powers/shriek.dm

```diff
@@ -11,7 +11,11 @@
 	for(var/mob/living/M in get_hearers_in_view(4, user))
 		if(iscarbon(M))
 			if(!M.mind || !M.mind.changeling)
-				M.adjustEarDamage(0,30)
+				if(iscarbon(M))

```

Comment:
There is already iscarbon check up there, also why not just adjustEarDamage here ?